src/index.jsx

```diff
@@ -1,3 +1,4 @@
+import 'babel-polyfill';  // general ES2015 polyfill (e.g. promise)
```

Comment:
This is basically what is done in Insights, https://github.com/edx/edx-analytics-dashboard/commit/58334e787f57f700dfe5b576b75515df4cac421a.